### ISSUE TITLE
[Bug]: Accessibility. Add new rule and remove selected rule buttons are not accessible with keyboard

### ISSUE THREAD

**Issue by sevapopov2** [VirtualBox/virtualbox#490]
### Version

trunk/main

### Host OS Type

Windows

### Host OS name + version

Windows 11 25H2 (OS Build 26200.7623)

### Host Architecture

x86

### Guest OS Type

Not applicable

### Guest Architecture

Not applicable

### Guest OS name + version

Not applicable

### Component

GUI

### What happened?

Add new rule and remove selected rule buttons in VM network portforwarding settings are not accessible with keyboard navigation.

### How can we reproduce this?

Precondition:
There is at least one VM in VM's list.

Steps to reproduce:

1. Open VM's settings.
2. Navigate to "Port forwarding" button on "Adapter 1" tab.
3. Navigate between the rules table and other elements with Tab key.

Actual behavior:
"Add new rule" and "Remove Selected Rule" buttons are not accessible with keyboard navigation.

Expected behavior:
Buttons are accessible and can be navigated with Tab key.

Reproduced on:
VirtualBox version 7.2.97-172127.

### Did you upload *all* of your necessary log files, screenshots, etc.?

- [x] Yes, I've uploaded all pertinent files to this issue.
